validator_client/src/cli.rs

```diff
@@ -306,9 +306,19 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                          for payload construction, where a strict fee recipient check will still be applied.")
                 .takes_value(false),
         )
+        .arg(
+            Arg::with_name("builder-registration-pubkey-override")
+                .long("builder-registration-pubkey-override")
+                .value_name("BUILDER_REGISTRATION_PUBKEY_OVERRIDE")
+                .alias("builder-registration-pubkey-override")
+                .help("This flag takes a 0x-prefixed hex public key that will be used to override the \
+                    pubkey used in builder api registration")
```

Comment:
```suggestion
                    pubkey used in builder api registration. This is useful for DVT setups.")
```

Just a suggestion to remind users (and developers) what this is for :)